camelot/view/action_steps/item_view.py

```diff
@@ -66,8 +65,11 @@ class Sort( ActionStep, DataclassSerializable ):
     @classmethod
     def gui_run(cls, gui_context, serialized_step):
         step = json.loads(serialized_step)
-        if gui_context.item_view != None:
```

Comment:
zou het nt de moeite zijn om een gui_contex.get_item_model() of zo toe te voegen ?

Reply:
Done. (dat is inderdaad handiger)